libraries/wallet/include/golos/wallet/wallet.hpp

```diff
@@ -45,6 +45,29 @@ namespace golos { namespace wallet {
             fc::optional<asset> min_delegation;
         };
 
+        struct private_inbox_query {
+            fc::flat_set<std::string> select_from;
+            std::string start_date;
+            fc::optional<bool> unread_only;
+            fc::optional<uint16_t> limit;
+            fc::optional<uint32_t> offset;
+        };
+
+        struct private_outbox_query {
```

Comment:
Can we just combine `private_inbox_query` and `private_outbox_query` to one structure, renaming `select_from` and `select_to` to something neutral like `accounts`? And the same with `inbox_query` and `outbox_query`.
This will allow to put initialisation logic (`private_xx_box_query` → `xx_box_query`) and maybe something else like filtering, into member function and remove code duplication.

Reply:
Ok. I combine two queries in one.